moveit_ros/move_group/src/default_capabilities/query_planners_service_capability.cpp

```diff
@@ -73,94 +73,98 @@ void MoveGroupQueryPlannersService::initialize()
 bool MoveGroupQueryPlannersService::queryInterface(
     const std::shared_ptr<rmw_request_id_t>& /* unused */,
     const std::shared_ptr<moveit_msgs::srv::QueryPlannerInterfaces::Request>& /*req*/,
-    const std::shared_ptr<moveit_msgs::srv::QueryPlannerInterfaces::Response>& res)
+    const std::shared_ptr<moveit_msgs::srv::QueryPlannerInterfaces::Response>& /* unused */)
 {
-  for (const auto& planning_pipelines : context_->moveit_cpp_->getPlanningPipelines())
-  {
-    const auto& pipeline_id = planning_pipelines.first;
-    const auto& planning_pipeline = planning_pipelines.second;
-    const planning_interface::PlannerManagerPtr& planner_interface = planning_pipeline->getPlannerManager();
-    if (planner_interface)
-    {
-      std::vector<std::string> algs;
-      planner_interface->getPlanningAlgorithms(algs);
-      moveit_msgs::msg::PlannerInterfaceDescription pi_desc;
-      pi_desc.name = planner_interface->getDescription();
-      pi_desc.pipeline_id = pipeline_id;
-      planner_interface->getPlanningAlgorithms(pi_desc.planner_ids);
-      res->planner_interfaces.push_back(pi_desc);
-    }
-  }
-  return true;
+  // TODO(sjahr): This is currently not working. Need to decide whether to fix or remove this.
```

Comment:
Where is this used currently?

Reply:
Never mind, looks like the RVIZ plugins need this :see_no_evil: